app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,154 @@
-// Code your JavaScript / jQuery solution here
+const WINNING_COMBOS = [
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [0, 4, 8],
+  [6, 4, 2]
+]
+
+var turn = 0;
+var currentGame = 0;
+
+function player() {
```

Comment:
Consider using an arrow function along with a ternary for this function to make it a little more concise
`const player = () => turn % 2 === 0 ? 'X' : 'O'`